files/class-meta-updater.php

```diff
@@ -82,7 +83,8 @@ public function get_max_id(): int {
 
 		global $wpdb;
 
-		$this->max_id = (int) $wpdb->get_var( 'SELECT ID FROM ' . $wpdb->posts . ' ORDER BY ID DESC LIMIT 1' );
+		// phpcs:ignore WordPress.DB.DirectDatabaseQuery.DirectQuery, WordPress.DB.DirectDatabaseQuery.NoCaching
+		$this->max_id = (int) $wpdb->get_var( "SELECT MAX(ID) FROM {$wpdb->posts}" );
```

Comment:
Since the intent of this PR is for code style changes and this is a different query (albeit, returning the same result), we should keep the original query and move the change to a new PR.

Reply:
Fixed in 34509a2ca